papaparse.js

```diff
@@ -415,6 +415,50 @@
 		}
 	}
 
+	function DynamicTypingConfig(config)
+	{
+		this._columnConfig = {};
+		this._isAllDynamic = false;
+		this._isBlackListOnly = true;
+
+		if (config === true)
+		{
+			this._isAllDynamic = true;
+			return;
+		}
+		else if (!config)
+		{
+			return;
+		}
+
+		if (config.__only instanceof Array)
+		{
+			this._isBlackListOnly = false;
+			config.__only.forEach(function(field)
```

Comment:
See [here](https://github.com/vyrak/PapaParse/blob/98dc2405a72c4598a599f96c946c61758a070e55/tests/test-cases.js#L664) for example of old hash config with specific columns configured. Notice how it's possible that a header can be named `'only'` or `'except'`; not likely, but possible.

If you scroll down [further](https://github.com/vyrak/PapaParse/blob/98dc2405a72c4598a599f96c946c61758a070e55/tests/test-cases.js#L682), you can see an example with the `'__parsed_extra'` working with specific columns configured.